etcdmain/etcd.go

```diff
@@ -89,6 +89,11 @@ func startEtcdOrProxyV2() {
 	defaultHostOverride := defaultHost == "" || defaultHostErr == nil
 	if (defaultHostOverride || cfg.Name != embed.DefaultName) && cfg.InitialCluster == defaultInitialCluster {
```

Comment:
@heyitsanthony Just moved them all to `embed.UpdateDefaultClusterFromName`. PTAL. Thanks.